cypress/e2e/RoundTrips/TemplateProject.feature

```diff
@@ -0,0 +1,202 @@
+    And  I expect '[data-cy="template-card_terraform_java_webapp"]' appear 1 time on screen
+    And  I expect '[data-cy="template-card_github_CI"]' not exists
+
+    # Clear filter text and verify all templates are present
+    When I set on '[data-cy="search-template-input"]' text ' '
+    Then I expect '[data-cy="template-card_terraform_webapp"]' appear 1 time on screen
+    And  I expect '[data-cy="template-card_terraform_java_webapp"]' appear 1 time on screen
+    And  I expect '[data-cy="template-card_github_CI"]' appear 1 time on screen
+
+    # Select Githubator-plugin plugin and verify all templates disappear
+    When I select '[data-cy="item_Github"]' in '[data-cy="language-select"]'
+    Then I expect '[data-cy="template-card_terraform_webapp"]' not exists
+    And  I expect '[data-cy="template-card_terraform_java_webapp"]' not exists
+    And  I expect '[data-cy="template-card_github_CI"]' appear 1 time on screen
+
+    # Select Terrator-plugin plugin and verify only all terraform templates are present
+    When I select '[data-cy="item_Terraform"]' in '[data-cy="language-select"]'
+    Then I expect '[data-cy="template-card_terraform_webapp"]' appear 1 time on screen
+    And  I expect '[data-cy="template-card_terraform_java_webapp"]' appear 1 time on screen
+    And  I expect '[data-cy="template-card_github_CI"]' not exists
+
+    ## 400 Add diagram template should redirect to draw page with correct plugin
+    When I click on '[data-cy="template-card_terraform_webapp"]'
+    And  I click on '[data-cy="submit-button"]'
+    Then I expect '[data-cy="import-model-template-dialog"]' exists
+
+    When I set on '[data-cy="import-model-template-form"] [data-cy="name-input"]' text '{{ diagramFolder }}/'
+    And  I click on '[data-cy="import-model-template-form"] [data-cy="submit-button"]'
+    Then I expect current url is '{{ projectName }}/modelizer/draw\?plugin=terrator-plugin&path={{ diagramFolder }}'
+    And  I expect '[data-cy="component-definitions-list"]' exists
+    And  I expect '[data-cy="component-definitions-item_terrator-plugin"] [data-cy="title"]' is 'terrator-plugin'
+
+    ## 401 Add diagram template should create diagram folders
+    When I click on '[data-cy="modelizer-switch-button"] [aria-pressed="false"]'
+    Then I expect current url is '{{ projectName }}/modelizer/text\?plugin=terrator-plugin&path={{ diagramFolder }}'
+    And  I expect '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]' is '{{ projectName }}'
+    And  I expect '[data-cy="file-explorer"] [data-cy="folder_{{ diagramFolder }}"]' exists
+    And  I expect '[data-cy="file-explorer"] [data-cy="file_{{ diagramFile }}"]' exists
+    And  I expect '[data-cy="active-tab"] [data-cy="file_{{ diagramFile }}"]' exists
+    And  I expect active file content to be equal to "cypress/resources/main.tf"
+
+    # Back to models page
+    When I click on '[data-cy="modelizer-switch-button"] [aria-pressed="false"]'
```

Comment:
not needed

Reply:
I forgot we have access to the button from the Text view 🫣